app/players/dwarves/dwarf.py

```diff
@@ -0,0 +1,12 @@
+from abc import ABC

```

Comment:
I think we don't need to import ABC here

Reply:
Why not? I implement abstract class and import module for it